modules/ROOT/pages/scopes-and-collections-config.adoc

```diff
@@ -0,0 +1,191 @@
+= Scopes and Collections Configuration for Sync Gateway
+:page-aliases: learn/scopes-and-collections-config.adoc
+ifdef::show_edition[:page-edition: {release}]
+ifdef::prerelease[:page-status: {prerelease}]
+:page-role:
+:page-content: conceptual
+:description: pass:q[How to configure Scopes and Collections for Sync Gateway - Examples with descriptions.]
+
+For a successful configuration of your Couchbase Sync Gateway Scopes and Collections, use this helpful example. 
+First, it's important to remember that buckets can hold multiple scopes, and each scope can contain multiple collections.
+
+When you create a bucket, a `bucket._default._default` element is inherently included, as per the documentation on xref:server/learn/data/scopes-and-collections.adoc#default-scope-and-collection[Default Scopes and Collections] in Couchbase Server.
+
+The default is unique to the bucket creation process and predominantly serves the purpose of backward compatibility.
+
+Please be aware that only Couchbase Server (CBS) allows using the word 'default' in naming conventions. 
+This explains why you may see the term _default.default when creating a new bucket. 
+Couchbase encourage you to explore and experiment with this concept within your Couchbase instance for practical experience.
+
+== Configuration Options
+
+You have these options to configure Scopes and Collections Sync for Gateway:
+
+* *custom scope/custom collection (the option we cover here)* +
+Gain maximum flexibility in organizing and accessing your data by customizing it to suit your needs.
+
+[NOTE]
+====
+Scopes is not required for upgrading to Sync Gateway 3.1. 
```

Comment:
Scopes are not required...

this note seems a little out of place. Move it after the bulleted list?